recipes/onnxruntime/all/test_package/conanfile.py

```diff
@@ -0,0 +1,23 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class OnnxRuntimeTestConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake", "cmake_find_package"
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def test(self):
+        tools.download(
+            "https://github.com/microsoft/onnxruntime/raw"
+            "/master/csharp/testdata/squeezenet.onnx",
+            "squeezenet.onnx"
+        )
```

Comment:
Not reproducible, downloads outside of `source()` should be avoided.

Reply:
source() doesn't work for `test_package` for some reason

Reply:
I think, we can create much more simple .onnx graph about ~10kb, which can be stored in the folder, i.e. like it is done in some packages for locale checking etc